docs/site/pages/content/assets-views.mdx

```diff
@@ -45,7 +45,7 @@ The `id` of the views are used in the navigation section to reference a specific
 
 ## Cross-field validation
 
-The other special property of a `view` vs. an `asset` is the addition of a `validation` property on the view. These contains [`validation` objects](../guides/validation) that are used for validations crossing multiple fields, and are ran on user navigation rather than data change.
+The other special property of a `view` vs. an `asset` is the addition of a `validation` property on the view. These contain [`validation` objects](../plugins/common-types#validations) that are used for validations crossing multiple fields, and are ran on user navigation rather than data change.
```

Comment:
i think this should be referencing https://github.com/player-ui/player/blob/main/docs/site/pages/content/schema.mdx#validation rather than validations from a plugin

Reply:
It doesn't seem like I can do a relative link for this (`../content/schema#validation`). Is it fine if I use the full link: `https://github.com/player-ui/player/blob/main/docs/site/pages/content/schema.mdx#validation`?

Reply:
can you just do `./schema#validation` since its also in the `content` folder?

Reply:
Oh yeah, sorry. My bad. That also doesn't work.